code/game/objects/items/theft_items.dm

```diff
@@ -0,0 +1,291 @@
+//Items for nuke theft, supermatter theft traitor objective
+
+
+// STEALING THE NUKE, this is disabled currently, as I don't think we need an objective to steal the nuclear device. The items work fine, just you wont be able to screw the nuke, get the core, nor does it have a core in it
+
+//the nuke core - objective item
+/obj/item/nuke_core
+	name = "plutonium core"
+	desc = "Extremely radioactive. Wear goggles."
+	icon = 'icons/obj/nuke_tools.dmi'
+	icon_state = "plutonium_core"
+	item_state = "plutoniumcore"
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF
+	var/pulse = 0
+	var/cooldown = 0
+	var/pulseicon = "plutonium_core_pulse"
```

Comment:
Undocumented change?

Reply:
> Also adds the items for the plutonium core objective that TG has, as they are in the same file and are the main type the supermatter steal items are based off of, but it is not added, as I did not feel an objective that disables the stations nuke is a good idea.

I can add it as an objective, I did not feel it was a good idea. If you wish, I can remove the plutonium core, screwdriver, and base case, but this will make it a little bit harder for someone to port later, if someone wishes to do that.

Reply:
Either port it or don't. Don't leave unused code in the codebase.

Reply:
Added